integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/StackDeleteInstanceAction.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.it.cloudbreak.newway.action;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.StackEntity;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+public class StackDeleteInstanceAction implements ActionV2<StackEntity> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StackDeleteInstanceAction.class);
+
+    @Override
+    public StackEntity action(TestContext testContext, StackEntity entity, CloudbreakClient client) throws Exception {
+        String instanceId = testContext.getSelected("instanceId");
```

Comment:
Whenever we are writing an Action that relies on a select command we should tell the framework user a hint like in Mockito.
Tried to run StackDeleteInstanceAction but no instanceId was selected.
Try to use `.select(s -> entity.getInstanceId(), key("instanceId"))`
Other suggestion is maybe if an Action requires a select, we should provide a Key String value based on the action.